.gitattributes

```diff
@@ -0,0 +1,24 @@
+#
+# Exclude these files from release archives.
+# This will also make them unavailable when using Composer with `--prefer-dist`.
+# If you develop for this plugin using Composer, use `--prefer-source`.
+# https://www.reddit.com/r/PHP/comments/2jzp6k/i_dont_need_your_tests_in_my_production
+# https://blog.madewithlove.be/post/gitattributes/
+#
+.gitattributes export-ignore
+.gitignore     export-ignore
+/.github       export-ignore
+/c             export-ignore
+/tests         export-ignore
```

Comment:
The `.gitattributes` file is unrelated to GitHub Actions. It's about keeping the development related files outside of other people's production environment (where they don't belong and could potentially even create a security risk, as has happened before).

Please read the articles which I linked to in the documentation of the file.
* https://www.reddit.com/r/PHP/comments/2jzp6k/i_dont_need_your_tests_in_my_production
* https://blog.madewithlove.be/post/gitattributes/

Reply:
Sure I know what `.gitattributes` is, and we use it in some other repos here. Good point about test files being a security risk for PHP projects in particular, where the files could commonly end up directly accessible via a web server. OK, let's do it the way you propose. However, then there's a new decision to make: should future phpass official tarballs be made with `git archive` (which I thought was the intent here) or maybe not (so they retain what they used to have, with my expectation being that only the one needed source file would be taken out and incorporated in a project's tree)?

Reply:
Well, rather than GPG-sign an archive downloaded from GitHub, I'd review my local copy and use `git archive` and sign that. Anyway, I could just be tarring up the whole tree excluding `.git` for such purpose, and people who don't care to have signed content can use the GitHub-generated archives that would also exclude tests.